pom.xml

```diff
@@ -15,12 +15,12 @@
                 this result is useless to us -->
         <findbugs.failOnError>false</findbugs.failOnError>
 
-		<selenium.version>3.12.0</selenium.version>
-		<htmlunit.version>2.29.2</htmlunit.version>
+		<selenium.version>3.141.59</selenium.version>
+		<htmlunit.version>2.33.3</htmlunit.version>
 		<java.level>8</java.level>
 	</properties>
 	<artifactId>selenium</artifactId>
-	<version>3.12.1-SNAPSHOT</version>
```

Comment:
The plugin version will be updated by the maven release plugin. It should not be updated manually.

Reply:
Reverted to previosu value